kentikapi/client.go

```diff
@@ -55,37 +78,46 @@ type Config struct {
 	CloudExportAPIURL string
 	// SyntheticsAPIURL defaults to "https://synthetics.api.kentik.com".
 	SyntheticsAPIURL string
-	AuthEmail        string
-	AuthToken        string
-	RetryCfg         RetryConfig
+	// SyntheticsGRPCHostPort defaults to "synthetics.api.kentik.com:443".
+	SyntheticsGRPCHostPort string
+	// CloudExportGRPCHostPort defaults to "cloudexport.api.kentik.com:443".
+	CloudExportGRPCHostPort string
+	AuthEmail               string
+	AuthToken               string
+	// RetryCfg has no effect on CloudExportAdmin, SyntheticsAdmin and SyntheticsData.
+	RetryCfg RetryConfig
```

Comment:
Gee, I thought that implementing retry strategy is the main point of the PR :-/.

Reply:
@Opelord This comment needs to be removed

Reply:
Yes, it will be changed in the next PR